src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1910,6 +1910,11 @@ bool Matcher::match_rule_supported(int opcode) {
     case Op_EncodeISOArray:
       return UseRVV;
 
+    case Op_RoundVF:
+      return UseRVV && MaxVectorSize >= 32;
+    case Op_RoundVD:
+      return UseRVV && MaxVectorSize >= 64;
```

Comment:
It would be worth leaving the same comment you've in the PR description here as well, to make it very clear in the sources why the option is enable/disabled on these parameters.

Reply:
Added comments. Thanks!